ansible/roles/test/tasks/arp_clear.yml

```diff
@@ -0,0 +1,140 @@
+######################################################################
+## This playbook tests that we are able to clear ARP entries using the
+## sonic-clear utility.
+######################################################################
+
+- name: Define some common test values
+  set_fact:
+    dut_ip: "10.10.1.2"
+    dut_prefixlen: "28"
+    arp_test_data:
+      - fake_mac: "00:06:07:08:09:00"
+        fake_ip: "10.10.1.3"
+      - fake_mac: "00:06:07:08:09:01"
+        fake_ip: "10.10.1.4"
+      - fake_mac: "00:06:07:08:09:02"
+        fake_ip: "10.10.1.5"
+
+- block:
+    - name: gather interface information if not available
+      interface_facts:
+      when: ansible_interface_facts is not defined
+
+    - name: Store the indices for the ports that operationally 'up'
+      set_fact: candidate_ifaces="{{ candidate_ifaces|default([]) + [item.device] }}"
+      with_items: ansible_interface_facts.values()
+      when: item.active and (item.device | search('Ethernet'))
+
+    - name: Choose the first candidate interface for testing
+      set_fact:
+        test_interface: "{{ candidate_ifaces[0] }}"
+        test_interface_index: "{{ minigraph_port_indices[candidate_ifaces[0]] }}"
+
+    - name: find if interface1 in portchannel
+      set_fact:
+        po1: "{{ item.key }}"
+      when: test_interface in item.value['members']
+      with_dict: minigraph_portchannels
+
+    - name: move interface {{ test_interface }} out of {{ po1 }}
```

Comment:
In test Log:

TASK [test : move interface {{ test_interface }} out of {{ po1 }}] *************<<<<<Not Sure why args are not replaced here, plz have a look once. 
Saturday 07 September 2019  02:35:08 +0000 (0:00:00.069)       0:00:28.405 **** 
skipping: [lab-ignw-seastone-dut-li1]

TASK [test : bring Ethernet8 up] ***********************************************
Saturday 07 September 2019  02:35:08 +0000 (0:00:00.083)       0:00:28.489 **** 
skipping: [lab-ignw-seastone-dut-li1]